ext/couchbase.cxx

```diff
@@ -3517,7 +3517,7 @@ cb_Backend_document_lookup_in(VALUE self, VALUE bucket, VALUE scope, VALUE colle
             if (!resp_entry.value.empty()) {
                 rb_hash_aset(entry, value_property, cb_str_new(resp_entry.value));
             }
-            if (resp_entry.ec && resp_entry.ec != couchbase::errc::key_value::path_not_found) {
+            if (resp_entry.ec) {
```

Comment:
There is no need for this check now that C++ clears `path_not_found` status codes for exists specs